tests/Nelmio/Alice/Fixtures/Parser/Methods/PhpTest.php

```diff
@@ -110,4 +110,31 @@ public function testIncludeFiles()
         ];
         $this->assertEquals($expectedData, $data);
     }
+
+    public function testIncludeFilesOrder()
+    {
+        $data = $this->parser->parse(__DIR__.'/../../../support/fixtures/include_order.php');
+
+        $expectedData = [
+            'Foo' =>
+                [
+                    'foo' =>
+                        [
+                            'id' => 1,
+                            'text' => '<word()>',
+                        ],
+                ],
+            'Bar' =>
+                [
+                    'bar' =>
+                        [
+                            'id' => 1,
+                            'text' => '<@foo->text>'
+                        ],
+                ],
+        ];
+
+        $this->assertEquals($expectedData, $data);
+        $this->assertTrue($expectedData === $data);
```

Comment:
>Do you think it would be better to use assertEquals() + assertTrue() or assertSame() only ?

I think we can go for the later. If we want more details on the error for debug purposes we can change it locally for debugging :) That being said I don't have a strong opinion on that one.